frontend/src/components/Tree/Tree.jsx

```diff
@@ -141,12 +147,21 @@ function renderTree(dimensions, jsonData, svgRef, onNodeClick, onRightClick) {
     })
     .attr("stroke", "black")
     .attr("fill", "none")
-    .attr("opacity", 1);
+    //If the parent node is highlighted, also highlight the link
+    .attr("opacity", (link) => (link.source.data.opacity === 1 ? 1 : 0.25));
   svg.attr("viewBox", autoBox).node();
   return [nodeGroupEnterAndUpdate, enteringAndUpdatingLinks];
 }
 
-export function Tree({ jsonData, onNodeClick, onRightClick }) {
+/**
+ *
+ * @param jsonData: JSON data representing the tree structure
```

Comment:
Nice job for this doc, please also include a short description at the top. And also add something like this for the other functions.

Reply:
please see my comment in the other MR, i.e. I am wondering if this is just a c thing or is it convention to do it in pyhon, node etc since I have never seen it before

Reply:
Sorry I should have clarified in both this and the other MR, what I meant in this case was that you have your docstring with @param and @returns and so on. Just above the @param but within the same doc string, also add a short description of what the function does.

Reply:
Got it, fixed